web/src/engine/core-processor/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+SUBPROJECT_NAME=engine/core-processor
+
+. "${KEYMAN_ROOT}/web/common.inc.sh"
+. "${KEYMAN_ROOT}/resources/shellHelperFunctions.sh"
+
+# ################################ Main script ################################
+
+builder_describe "Keyman Core WASM integration" \
+  "@/core:wasm" \
+  "clean" \
+  "configure" \
+  "build" \
+  "test" \
+  "--ci+                     Set to utilize CI-based test configurations & reporting."
+
+builder_describe_outputs \
+  configure    "/web/src/engine/core-processor/src/import/core/core-interface.d.ts" \
+  build        "/web/build/${SUBPROJECT_NAME}/lib/index.mjs"
+
+builder_parse "$@"
+
+#### Build action definitions ####
+
+do_clean() {
+  rm -rf "${KEYMAN_ROOT}/web/build/${SUBPROJECT_NAME}"
+  rm -rf "src/import/"
+}
+
+do_configure() {
+  verify_npm_setup
+
+  mkdir -p "src/import/core/"
+  # we don't need this file here, but it's nice to have for reference and auto-completion
+  cp "${KEYMAN_ROOT}/core/build/wasm/${BUILDER_CONFIGURATION}/src/core-interface.d.ts" "src/import/core/"
```

Comment:
It is copied in `configure` so that it is available before build, during editing sessions (assuming one has done a `build.sh configure` before trying to work). This matches the dependency evaluation model we use, but of course if the interface changes. then `configure` has to be run again.